_posts/2025-03-03-GPU-Accelerated-Vector-Search-OpenSearch-New-Frontier.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "GPU-accelerated vector search in Opensearch: A new frontier" 
+authors:
+  - navneev
+  - cnolet
+  - kshitizgupta21
+  - dylantong
+  - nwstephens
+  - vamshin
+date: 2025-03-03
+has_science_table: true
+categories:
+  - technical-posts
+meta_keywords: Vector Database, Vector Engine, k-NN plugin,  OpenSearch 3.0, k-nn search, Vector Search, Semantics Search, Index Build
+meta_description: Learn about how OpenSearch Vector Engine is going to use GPU for accelerating its Vector Index Builds
+---
+
+OpenSearch's adoption as a vector database has grown significantly with the rise of generative AI applications. Vector search workloads are scaling from millions to billions of vectors, making traditional CPU-based indexing both time consuming and cost intensive. To address this challenge, OpenSearch is introducing GPU acceleration as a [preview feature](https://github.com/opensearch-project/k-NN/issues/2293) for the OpenSearch Vector Engine in the upcoming 3.0 release by using [NVIDIA cuVS](https://github.com/rapidsai/cuvs). By leveraging the massive parallel processing capabilities of GPUs, this new feature dramatically reduces index building time, significantly lowering operational costs while delivering superior performance for large-scale vector workloads.
```

Comment:
Please link the words "vector database" in the first sentence here: https://opensearch.org/platform/search/vector-database.html